Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -247,7 +247,7 @@ - (void)_updatePlaceholder
   self.attributedPlaceholder = [[NSAttributedString alloc] initWithString:self.placeholder ?: @""
                                                                attributes:[self _placeholderTextAttributes]];
 #else // [TODO(macOS ISS#2323203)
-  self.placeholderAttributedString = [[NSAttributedString alloc] initWithString:self.placeholder ?: @""
+  self.placeholderAttributedString = [[NSAttributedString alloc] initWithString:self.placeholderString ?: @""
```

Comment:
So this one is interesting, I think the compiler thinks it’s available due to `RCTUITextField` implementing the `RCTBackedTextInputViewProtocol` protocol, which has a `placeholder` property [here](https://github.com/microsoft/react-native-macos/blob/a87bd69ed6fb0e7d411087c19eec40114c246419/Libraries/Text/TextInput/RCTBackedTextInputViewProtocol.h#L37). However, that seems to rely on `UITextField` actually having a `placeholder` property, whereas `NSTextField` has `placeholderString`, which is also used elsewhere.

However, `placeholder` is still being used [here](https://github.com/microsoft/react-native-macos/blob/a87bd69ed6fb0e7d411087c19eec40114c246419/Libraries/Text/TextInput/Singleline/RCTUITextField.m#L491) and was also used in 0.61; what gives?

Reply:
In RCTUITextField there are `setPlaceholder:` and `placeholder` selectors defined.   The impls have #if/defs to make mac call super setPlaceholderString/placeholderString.   The setter was overridden upstream too.   Did they get removed in .62?

Reply:
Ya, it looks like the setPlaceholder: selector is still in the fork, but somehow during the merge our fork's addition of the getter `placeholder` got deleted.   It just needs to be restored to be after the `setPlaceholder:` in `RCTUITextField.m`:
```m
- (NSString*)placeholder // [TODO(macOS ISS#2323203)
{
#if !TARGET_OS_OSX 
  return super.placeholder;
#else
  return self.placeholderAttributedString.string ?: self.placeholderString;
#endif
} // ]TODO(macOS ISS#2323203)
```

Reply:
Aha, the pieces fall into place 👍